static/js/util/editEducation.js

```diff
@@ -41,9 +41,9 @@ export function openNewEducationForm(level: string, index: number) {
 }
 
 export function deleteEducationEntry () {
-  const { saveProfile, profile, ui, deletionIndex } = this.props;
+  const { saveProfile, profile, ui } = this.props;
```

Comment:
ah, yeah, in that case you should probably keep what's here. It doesn't make `ui` available (I think because `{ foo: { foo: 'bar' } }` in legal in JS, so it wouldn't be able to unambiguously deconstruct all cases like `{ foo: { foo } } = obj` and make both the outer and inner value available. If you wanted both you'd have to do this:

```js
const { saveProfile, profile, ui, ui: { deletionIndex } } = this.props;
```

which is IMO a little ugly